### ISSUE TITLE
keystroke package incompatible

### ISSUE THREAD

**Issue by mbertucci47** [latex3/tagging-project#265]
The [keystroke](https://www.ctan.org/pkg/keystroke) package produces parent-child and "missing alt text" warnings.
```tex
\DocumentMetadata
  {
    lang=en-US,
    pdfversion=2.0,
    pdfstandard=ua-2,
    testphase={phase-III,math,table,title,firstaid}
  }

\documentclass{article}
\usepackage{keystroke}

\begin{document}

\keystroke{A}

\end{document}
```
The included graphics should be artifacts. Not sure about the parent-child warnings.
